### ISSUE TITLE
Matrix/IRC/github discussions/etc support rooms

### ISSUE THREAD

**Issue by i509VCB** [iximeow/yaxpeax-x86#20]
I personally dislike spamming github's issue reports with questions about library usage and am wondering if there are any plans to open a place where we can ask questions without spamming github issues.

**Comment by iximeow** [comment]
i agree, github issues don't lend themselves to usage/support or even design conversation very much. elsewhere i end up using github issues (or comments therein) as a summary of a conversation that happens somewhere more interactive, for posterity.

i've been wondering what would be a good approach for yaxpeax generally. i don't love _requiring_ people to make an account with github (or elsewhere) to send me a note either. there's a few conversations that i've had with people over email, for example, and i'm pretty happy about that except for the parts where email is kind of awful.

discord is definitely not happening, irc is _okay_ but difficult for accessibility reasons, matrix is better but a little more cumbersome - i'd prefer not asking people to find a homeserver, and for usage questions (_hopefully_ the primary use) i'm not sure how to get a publicly-legible history.

so what i've been actually leaning towards is something more forum-like, Discourse maybe? zulip "works" but isn't as public as i'd like. i see that these days gitter is integrated with matrix, and gitter is legible without an account, so maybe that's actually the best choice for now?

i'll start actually giving some of this stuff a try in the next week or so, see what at least feels tolerable..

**Comment by iximeow** [comment]
gitter does seem to be like the least irritating option, and does bridge to matrix well enough. so, now there's https://gitter.im/yaxpeax/yaxpeax#, or `#yaxpeax_yaxpeax:gitter.im` to other matrix clients.

next up is actually writing that down in places people would expect to find it.
